src/containers/PageBuilder/SectionBuilder/SectionFooter/SectionFooter.module.css

```diff
@@ -0,0 +1,185 @@
+@import '../../../../styles/customMediaQueries.css';
+
+.footer {
+  /* padding: 32px;
+  max-width: calc(var(--contentMaxWidth) + 72px);
+  margin: 0 auto; */
+  max-width: var(--contentMaxWidthPages);
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 32px;
+  margin: 0 auto;
+  padding: 32px;
+}
+
+.footer ul {
+  list-style-type: none;
+  padding-inline-start: 0px;
+}
+
+.linkList {
+  list-style-type: none;
+  padding-left: 12px;
+}
+
+.content {
+  display: grid;
+  justify-content: center;
+  align-content: end;
+  grid-auto-rows: auto;
+  grid-gap: 32px;
+  grid-row-gap: 16px;
+  grid-template-columns: 1fr;
+  grid-template-areas: 'logo' 'details' 'grid' 'links';
+}
+
+.grid {
+  grid-area: grid;
+  display: grid;
+  justify-content: center;
+  align-content: end;
+  grid-auto-rows: auto;
+  grid-gap: 32px;
+}
+
+.gridCol1 {
+  grid-template-columns: 1fr;
+}
+
+.gridCol2,
+.gridCol3,
+.gridCol4 {
+  grid-template-columns: 1fr 1fr;
+}
+
+.detailsInfo {
+  grid-area: details;
+  & span {
+    font-size: 16px;
+  }
+}
+
+.logo {
+  grid-area: logo;
+}
+
+.socialInfo {
+  grid-area: links;
+  & span {
+    font-size: 12px;
+    line-height: 16px;
+    color: #6e6e73;
+  }
```

Comment:
This could also be a class: `.copyright`.

This is actually pretty error-prone: it means that any span (however deeply nested) will get these rules.

You also use BlockBuilder inside the same .socialInfo div. So, when someone customizes social media links in the future, they might add <span> element there too. In that kind of scenario, it's usually quite frustrating to debug why some specific rules are applied and how to take them off without breaking something else.

```
          <div className={css.socialInfo}>
            <div className={css.icons}>
              {showSocialMediaLinks ? <BlockBuilder blocks={linksWithBlockId} /> : null}
            </div>
            <Field data={copyright} />
          </div>
```

Reply:
Thanks, fixed!